core/src/main/resources/mappings/scheduled-jobs.json

```diff
@@ -663,6 +663,29 @@
           "enabled": false
         }
       }
+    },
+    "workflow_metadata" : {
+      "properties": {
+        "workflow_id": {
+          "type": "keyword"
+        },
+        "monitor_ids": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
```

Comment:
256 is a small number
do we need this for a list?

Reply:
Yes it is